build-system/pr-check/e2e-tests.js

```diff
@@ -67,7 +67,7 @@ async function main() {
     ) {
       downloadDistOutput(FILENAME);
       timedExecOrDie('gulp update-packages');
-      timedExecOrDie('gulp e2e --nobuild --headless --compiled');
+      timedExecOrDie('gulp e2e --nobuild --headless --compiled --coverage');
```

Comment:
This should eventually be gated with `argv.coverage`; likewise with the `gulp dist`/`gulp build` commands below, which I had just enabled `--coverage` on while experimenting/trying to get it to run on Travis.

Reply:
Agree. Adding an additional job to the CI matrix isn't that expensive, so we should probably just do all coverage measurements in a parallel job.